packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -642,9 +637,7 @@ class SemanticsData with Diagnosticable {
   }
 
   @override
-  int get hashCode {
-    return ui.hashValues(
-      ui.hashValues(
+  int get hashCode => Object.hashAll(<Object?>[
```

Comment:
why are we introducing a list here?

Reply:
The number of items is >20. I supposed having a list would be better than nested `Object.hash`s. If you think otherwise, I'll change it back.

Reply:
lists need to be allocated, which causes GC pressure, etc. I think it's more efficient to just nest Object.hash calls.

Reply:
Will update and keep that in mind. Thanks (: